src/harness/fake_sas.py

```diff
@@ -177,14 +177,15 @@ def _GetMissingParamResponse(self):
     return {'responseCode': MISSING_PARAM}
 
   def InjectEscZone(self, request, ssl_cert=None, ssl_key=None)
-    zoneData = req['zoneDate']
-    if(zoneData['usage'] == 'EXCLUSION_ZONE'):
-      idAttributes = zoneData['id'].split('/')
-      zoneId = ''; 
-     for zoneIdPart in idAttributes[4 : len(idAttributes)]:
-       zoneId+=zoneIdPart
-     return {'zone_id' : zoneId}
-		
+    zone_data = request['zoneData']
+    if(zone_data['usage'] == 'EXCLUSION_ZONE'):
```

Comment:
Why not just return the zone ID directly? I don't think it's necessary to parse and then recombine the ID.